src/fstext/trivial-factor-weight.h

```diff
@@ -52,17 +52,8 @@
 // This has the advantage that it always works, for any input (also I just
 // prefer this approach).
 
-#ifdef _MSC_VER
 #include <unordered_map>
 using std::unordered_map;
```

Comment:
there is a header ./util/stl-utils.h that would do the same.  Not sure what would be the prefered solution, though.